pkgs/peripheryapp/periphery/registry.yaml

```diff
@@ -4,13 +4,11 @@ packages:
     repo_owner: peripheryapp
     repo_name: periphery
     description: A tool to identify unused code in Swift projects
-    version_constraint: "false"
+    version_constraint: semver("> 2.13.0")
+    format: zip
+    asset: periphery-{{.Version}}.{{.Format}}
     version_overrides:
       - version_constraint: semver("<= 2.13.0")
```

Comment:
Please follow the style guide of version_constraint and version_overrides.

https://aquaproj.github.io/docs/develop-registry/registry-style-guide#version_overrides-style-guide

Reply:
Fixed. 80c7b8b55302c8084a21506e64ad046f8d6c41f2